website/content/en/docs/ansible/development-tips.md

```diff
@@ -6,107 +6,115 @@ weight: 12
 This document provides some useful information and tips for a developer
 creating an operator powered by Ansible.
 
-## Getting started with the k8s Ansible modules
+## Getting started with the Kubernetes Collection for Ansible
 
 Since we are interested in using Ansible for the lifecycle management of our
-application on Kubernetes, it is beneficial for a developer to get a good grasp
-of the [k8s Ansible module][k8s_ansible_module]. This Ansible module allows a
-developer to either leverage their existing Kubernetes resource files (written
-in YaML) or express the lifecycle management in native Ansible. One of the
-biggest benefits of using Ansible in conjunction with existing Kubernetes
-resource files is the ability to use Jinja templating so that you can customize
-deployments with the simplicity of a few variables in Ansible.
+application on Kubernetes, it is beneficial for a developer to get a good
+grasp of the [Kubernetes Collection for Ansible](https://galaxy.ansible.com/community/kubernetes).
+This Ansible collection allows a developer to either leverage their existing
+Kubernetes resource files (written in YAML) or express the lifecycle
+management in native Ansible. One of the biggest benefits of using Ansible in
+conjunction with existing Kubernetes resource files is the ability to use
+Jinja templating so that you can customize deployments with the simplicity of
+a few variables in Ansible.
 
-The easiest way to get started is to install the modules on your local machine
-and test them using a playbook.
+The easiest way to get started is to install the collection on your local
+machine and test them using a playbook.
 
-### Installing the k8s Ansible modules
+### Installing the Kubernetes Collection for Ansible
+
+To install the Kubernetes Collection, one must first install Ansible 2.9+.
+For example, on Fedora/Centos:
```

Comment:
I understand that it is about https://docs.ansible.com/ansible/latest/modules/k8s_module.html. Am I right? Could we also add the links and clarify that the following steps is specifically about the collection k8s?

Reply:
No. We have switched over to the [Kubernetes Collection for Ansible](https://galaxy.ansible.com/community/kubernetes) which provides other modules in addition to `k8s_module`. This new link is added in the revised doc.

Reply:
Can we please add the info and the link for it?